{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2020-4302",
			"ASSIGNER": "cve@mitre.org"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/176610",
					"name": "ibm-cognos-cve20204302-code-exec (176610)",
					"refsource": "XF",
					"tags": []
				},
				{
					"url": "https://www.ibm.com/support/pages/node/6346922",
					"name": "https://www.ibm.com/support/pages/node/6346922",
					"refsource": "CONFIRM",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "IBM Cognos Analytics 11.0 and 11.1 could allow a remote attacker to execute arbitrary code on the system, caused by a CSV injection. By persuading a victim to open a specially-crafted excel file, an attacker could exploit this vulnerability to execute arbitrary code on the system. IBM X-Force ID: 176610."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2020-10-12T14:15Z",
	"lastModifiedDate": "2020-10-13T10:55Z"
}